.github/workflows/ci.yml

```diff
@@ -0,0 +1,103 @@
+name: Continuous Integration
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types: [ opened, synchronize, reopened, ready_for_review ]
+    paths-ignore:
+      - 'config/**'
+      - 'docker/**'
+      - 'docs/**'
+      - 'images/**'
+      - 'scripts/**'
+      - 'tests/**'
+      - '**.md'
+      - '.gitignore'
+
+env:
+  GO111MODULE: on
+  GONOSUMDB: "*"
+  GOSUMDB: off
+
+jobs:
+  CommitLint:
+    name: commit lint
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 100
+      - name: Run commitlint
+        uses: wagoid/commitlint-github-action@v5
+      - name: Run licence check
+        run: make licence-check
+  StaticCheck:
+    name: "static check"
+    needs: CommitLint
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v1
+        with:
+          fetch-depth: 1
+      - uses: WillAbides/setup-go-faster@v1.7.0
+        with:
+          go-version: "1.19"
+      - name: Run static check
+        run: make static-check
```

Comment:
There is no Makefile that contains the command. The following steps are also similar.